pkgs/os-specific/linux/libsepol/default.nix

```diff
@@ -6,25 +6,31 @@ stdenv.mkDerivation rec {
   se_release = "20170804";
   se_url = "https://raw.githubusercontent.com/wiki/SELinuxProject/selinux/files/releases";
 
+  outputs = [ "bin" "out" "dev" "man" ];
```

Comment:
It would be nice to split the `static` library (1.5M) out but I am not sure how to do it without breaking the pkgconfig use.